src/software/jetson_nano/systemd/announcement.service

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description=announcement service
+
+# When systemd stops or restarts the thunderbots.service, the action is propagated to this unit
+PartOf=thunderbots.service
+
+# At startup, start this unit after the thunderbots.service start
+After=thunderbots.service thunderloop.service network.target
+
+[Service]
+# Pretend that the component is running
+# todo: execute announcement binary
```

Comment:
maybe create an overarching ticket here for all of these todos and tag the ticket number

Reply:
Added ticket and fixed formatting